### DISCUSSION TITLE
Automatic turn off in advanced mode

### DISCUSSION THREAD

**Discussion by bramborman** [pcolby/nfc-quick-settings#48] [Ideas]
I'm not sure how other people use this app, but my usage pattern is usually turn NFC on, pay, turn NFC off. I turn the NFC off almost immediately after the payment, if I don't forget 😅
It would be really helpful if I could select a time after which it would be automatically turned off after turning it on. That way I wouldn't have to worry about turning it off myself and would be free of the threat of forgetting 😁
This would, of course, be possible only with the advanced mode and ideally be configurable in a popup (#49).

**Comment by corobin** [comment]
this didn't even occur to me but it does sound very helpful now that you mention it

**Comment by SirNickolas** [comment]
Exact same workflow! I currently have it implemented via LlamaLab’s Automate, but it requires granting `WRITE_SECURE_SETTINGS` to a closed-source app, which I’m not really happy with. That’s why I’m looking for a tiny open-source alternative, and this application looks like it would be an ideal fit.
